### PR TITLE
feat: updated for fsc git action

### PR DESCRIPTION
**Pull request by ozayr-zaviar**
## Summary
- Git actions updated to run FSC on git actions.

## Note
- Need to change travisci-tools branch from `mnoman/gitActions` to `master`

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
         token: ${{ secrets.CI_USER_TOKEN }}
         repository: 'optimizely/travisci-tools'
         path: 'home/runner/travisci-tools'
-        ref: 'master'
+        ref: 'mnoman/gitActions'
```
Comment (msohailhussain) [inline .github/workflows/integration_tests.yml:21]:
```suggestion
        ref: 'master'
```

File: .github/workflows/swift.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
 
   integration_tests:
     if: "${{ github.event.inputs.PREP == '' && github.event.inputs.RELEASE == '' }}"
-    uses: optimizely/swift-sdk/.github/workflows/integration_tests.yml@master
+    uses: optimizely/swift-sdk/.github/workflows/integration_tests.yml@uzair/test-with-fsc
```
Comment (jaeopt) [inline .github/workflows/swift.yml:29]: same here?  should be @master ?

**Review by jaeopt — CHANGES_REQUESTED** [review]
a fix suggested

**Review by msohailhussain — APPROVED** [review]
lgtm

**Review by jaeopt — APPROVED** [review]

**Pull request merged**